src/test/java/org/alliancegenome/curation_api/VariantFmsITCase.java

```diff
@@ -0,0 +1,218 @@
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "MR_05_no_genomic_reference_sequence.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "MR_06_no_genomic_variant_sequence.json", params);
+
+		params.put("Entities", createCountParams(1, 0, 1, 0));
+		params.put("Locations", createCountParams(1, 0, 1, 0));
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "MR_07_no_allele_id.json", params);	
+	}
+	
+	@Test
+	@Order(4)
+	public void variantFmsBulkUploadEmptyRequiredFields() throws Exception {
+		HashMap<String, HashMap<String, Integer>> params = new HashMap<>();
+		params.put("Entities", createCountParams(1, 1, 0, 0));
+		params.put("Locations", createCountParams(1, 1, 0, 0));
+		params.put("Associations", createCountParams(1, 1, 0, 0));
+		
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "ER_01_empty_sequence_of_reference_accession_number.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "ER_02_empty_type.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "ER_03_empty_genomic_reference_sequence.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "ER_04_empty_genomic_variant_sequence.json", params);
+
+		params.put("Entities", createCountParams(1, 0, 1, 0));
+		params.put("Locations", createCountParams(1, 0, 1, 0));
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "ER_05_empty_allele_id.json", params);	
+	}
+	
+	@Test
+	@Order(3)
+	public void variantFmsBulkUploadInvalidFields() throws Exception {
+		HashMap<String, HashMap<String, Integer>> params = new HashMap<>();
+		params.put("Entities", createCountParams(1, 1, 0, 0));
+		params.put("Locations", createCountParams(1, 1, 0, 0));
+		params.put("Associations", createCountParams(1, 1, 0, 0));
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "IV_01_invalid_type.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "IV_02_invalid_type_for_fms_submissions.json", params);
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "IV_03_invalid_consequence.json", params);
+
+		params.put("Entities", createCountParams(1, 0, 1, 0));
+		params.put("Associations", createCountParams(1, 0, 1, 0));
+		
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "IV_04_invalid_sequence_of_reference_accession_number.json", params);
+		
+		params.put("Locations", createCountParams(1, 0, 1, 0));
+		params.put("Associations", createCountParams(1, 1, 0, 0));
+		checkBulkLoadRecordCounts(variantFmsBulkPostEndpoint, variantFmsTestFilePath + "IV_05_invalid_allele_id.json", params);	
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace
<!-- __reviewdog__:ChA5NDcxYmFmOTRjM2M3Yzk3EgpjaGVja3N0eWxl -->